ors-benchmark/src/main/java/org/heigit/ors/coordinates_generator/generators/CoordinateGeneratorMatrix.java

```diff
@@ -208,29 +244,31 @@ private void shutdownExecutor(ExecutorService executor) {
      * Exemplary table below.
      *
      *     A           B           C           D
-     *   +-----------+-----------+-----------+-----------+
+     * +-----------+-----------+-----------+-----------+
      * 1 | 1->A      | 1->B      | 1->C      | 1->D      |
      *   | A->1      | B->1      | C->1      | D->1      |
-     *   +-----------+-----------+-----------+-----------+
+     * +-----------+-----------+-----------+-----------+
      * 2 | 2->A      | 2->B      | 2->C      | 2->D      |
      *   | A->2      | B->2      | C->2      | D->2      |
-     *   +-----------+-----------+-----------+-----------+
+     * +-----------+-----------+-----------+-----------+
      * 3 | 3->A      | 3->B      | 3->C      | 3->D      |
      *   | A->3      | B->3      | C->3      | D->3      |
-     *   +-----------+-----------+-----------+-----------+
+     * +-----------+-----------+-----------+-----------+
      * 4 | 4->A      | 4->B      | 4->C      | 4->D      |
      *   | A->4      | B->4      | C->4      | D->4      |
-     *   +-----------+-----------+-----------+-----------+
+     * +-----------+-----------+-----------+-----------+
      *
      * In a brute force approach, this would mean O(2 * X * Y) routing checks.
-     * We use a chaining approach instead, which can reduce complexity to O(2 * (X + Y) + 2):
-     *  If all column points are reachable (A->B->C->D)
-     *  and all row points are reachable in reverse (4->3->2->1)
-     *  and we find a connection (1->A)
-     *  then we ensure all combinations have at least one connection by transitivity.
-     *  This guarantees full bidirectional connectivity across the matrix with linear effort,
-     *  leveraging structure over exhaustive search.
-     *  For the backwards case, we just connect last column to last row item.
+     * We use a chaining approach instead, which can reduce complexity to O(2 * (X +
+     * Y) + 2):
+     * If all column points are reachable (A->B->C->D)
+     * and all row points are reachable in reverse (4->3->2->1)
+     * and we find a connection (1->A)
+     * then we ensure all combinations have at least one connection by transitivity.
+     * This guarantees full bidirectional connectivity across the matrix with linear
+     * effort,
+     * leveraging structure over exhaustive search.
+     * For the backwards case, we just connect last column to last row item.
```

Comment:
I suspect there are potentially the following two issues with the above documentation which you might wnat to address.

1. Technical: in order for the table and the list of conditions to be rendered properly they might need to be formatted with the use of HTML tags.

2. Conceptual: if I'm not mistaken then the described procedure of checking the connections within X columns and the Y rows plus the two connection (here: 1->A and D->4) involves a total of `(X-1) + (Y-1) + 2` routing calls yielding a complexity of `O(X+Y)` rather than the mentioned ` O(2 * (X + Y) + 2)`.